flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolInteractionsTest.java

```diff
@@ -108,19 +109,18 @@ public void testCancelSlotAllocationWithoutResourceManager() throws Exception {
             // wait for the timeout of the pending slot request
             timeoutFuture.get();
 
-            assertEquals(0L, pool.getNumberOfWaitingForResourceRequests());
+            assertEquals(0L, pool.getAllocatedSlotsInformation().size());
```

Comment:
I think this is not the correct translation. We need to check that the pending slot request has been removed.